include/Jit/LLILCJit.h

```diff
@@ -174,6 +180,32 @@ class NullResolver : public llvm::RuntimeDyld::SymbolResolver {
   }
 };
 
+class ObjectLoadListener {
```

Comment:
I still think that this type should be moved into LLILCJit.cpp, as it's not used outside that file. Its presence here is only to satisfy the typedef on line 221--this whole definition can be replaced with a forward declaration (i.e. `class ObjectLoadListener;`), which is all that is necessary for the header file.